### ISSUE TITLE
Low accuracy on Teacher model. 

### ISSUE THREAD

**Issue by JingBo-L** [Jia-Research-Lab/MSAD#7]
Hi, This is a good work, the idea is very novel! The Student Model is ok,  but when I training on Teacher Model , its mAp is only around 20, the config files are R50-T.yaml and SLR100-50-T.yaml.  the batchsize was reduce to 4 ,and learning rate also reduce by 50%. Machine is 2x 3090. 

I want to know do you have similar  problems when you training?
Thanks

**Comment by qqlu** [comment]
I am not sure about that. 

In my experience, you can make sure that the R-50 ImageNet Pretrained model is rightly loaded.

**Comment by JingBo-L** [comment]
Thanks for your answer, I forget scale up the iteration number
